.pre-commit-config.yaml

```diff
@@ -60,4 +60,4 @@ repos:
     rev: v2.1.0
     hooks:
     -   id: codespell
-        args: ['--ignore-words-list','pullrequest']
+        args: ['--ignore-words-list','pullrequest,pullrequests']
```

Comment:
Otherwise:

```
codespell................................................................Failed
- hook id: codespell
- exit code: 65

.ci/bumpStackVersion.groovy:185: pullRequests ==> pull requests
.ci/bumpStackVersion.groovy:186: pullRequests ==> pull requests
.ci/bumpStackVersion.groovy:186: pullRequests ==> pull requests
.ci/bumpStackVersion.groovy:187: pullRequests ==> pull requests


```